bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java

```diff
@@ -47,9 +49,12 @@
  * @author Jochen Hiller - Initial contribution
  * @author Laurent Garnier - add support for all API languages
  * @author Laurent Garnier - add support for OGG and AAC audio formats
+ * @author cURLy bOi - add support for all API languages (again) and their respective voices
```

Comment:
Who's that???

Reply:
It iwas from the guy who started this enhancement (other PR).
Reoved.